bundles/net.tourbook.common/src/net/tourbook/common/util/IAdvancedMenuForActions.java

```diff
@@ -17,21 +17,21 @@
 
 public interface IAdvancedMenuForActions {
 
-	/**
-	 * Method is called when the advance menu is displayed
-	 */
-	void onShowMenu();
+   /**
+    * Method is called when the advance menu is displayed
+    */
+   void onShowMenu();
 
-	/**
-	 * Resets data which are keept when the advanced menu is open
-	 */
-	void resetData();
+   /**
+    * Resets data which are kept when the advanced menu is open
+    */
+   void resetData();

```

Comment:
When I do a cleanup of old code, then I also set the line delimiters to Unix

![image](https://github.com/mytourbook/mytourbook/assets/1283445/ff30791b-e0bc-4cfb-84f6-469790c09060)

Before

![image](https://github.com/mytourbook/mytourbook/assets/1283445/5637245a-b749-4579-9c49-e106334d4bee)

After

![image](https://github.com/mytourbook/mytourbook/assets/1283445/32e3ce83-164e-4c21-8580-9c9db90a9ab9)

Reply:
Good to know. Is it possible that this is automatically applied when saving a file like other actions ?

Reply:
I've not seen your requested feature in the 122 save actions